plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -442,7 +449,7 @@ func (c *nodePlugin) getForbiddenCreateLabels(modifiedLabels sets.String) sets.S
 }
 
 // getForbiddenLabels returns the set of labels that may not be set by the node on update.
```

Comment:
getForbiddenUpdateLabels ?

Reply:
I'll update this comment. Can you confirm the descriptions are correct?